src/Css.elm

```diff
@@ -4220,6 +4225,11 @@ color c =
 
 
 {-| -}
+mediaQuery : String -> List Snippet -> Snippet
+mediaQuery queryString snippets =
+  media [ customMediaQuery queryString ] snippets
```

Comment:
Can we change this to `media [ MediaQuery queryString ] snippets`? I think we can do without the helper function here! 😄